src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -507,9 +507,14 @@ std::vector<std::string> disabledTestPatterns() {
 #if !defined(OPENVINO_ARCH_ARM64)
     retVector.emplace_back(R"(smoke_Snippets.*)");
 #endif
```

Comment:
If we want to refactor `skip_test_config` for snippets, I would suggest to do the following in addition:
- Extract x64-related skipped tests to the separate body behind `if defined(OPENVINO_ARCH_X86_64)`
- Extract arm64-related skipped tests to the separate body behind `if defined(OPENVINO_ARCH_ARM64)`
- Also if the current platform is not x64 and not arm64, let's disable Snippets test at all: `R"(smoke_Snippets.*)"`

It should be clearer. What do you think?